.github/workflows/docs.yml

```diff
@@ -0,0 +1,38 @@
+on:
+  pull_request:
+    branches:
+      - v0
+name: docs
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Setup Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.8"
+    - name: Install nox
+      run: |
+        python -m pip install --upgrade setuptools pip wheel
+        python -m pip install nox
+    - name: Run docs
+      run: |
+        nox -s docs
+  docfx:
```

Comment:
The docfx check is failing. Should we remove it?

Reply:
Removed! PTAL